src/applications/vaos/tests/components/FacilityAddress.unit.spec.js

```diff
@@ -102,11 +102,7 @@ describe('VAOS Component: FacilityAddress', () => {
 });
 
 describe('VAOS Component: FacilityAddress when vaOnlineSchedulingPhysicalLocation is on', () => {
-  const initialState = {
-    featureToggles: {
-      vaOnlineSchedulingPhysicalLocation: true,
-    },
-  };
+  const initialState = {};
```

Comment:
Suggest changing the describe to something like "VAOS Component: FacilityAddress when clinic physical location is available" instead of the reference "FacilityAddress when vaOnlineSchedulingPhysicalLocation is on"

Reply:
Good callout. Thanks!